docs/QuickStart/HelmSetup.md

```diff
@@ -69,7 +69,7 @@ helm install devlake . \
     --set "ingress.tlsSecretName=ssl-certificate"
 ```
 
-After deployed, visit devlake: https://devlake-0.example.com:8443, and grafana at https://devlake-0.example.com:8443/grafana
```

Comment:
@keon94 Are you sure we don't need the port number here? If that's the case, do we also want to update line 56 of this file?

Reply:
Honestly, having played with it some more, the ingress.HttpsPort setting has been inconsequential on my machine. I used both 8443 and 8444. Turns out 8443 is coincidentally the target port on the kube apiserver, and if I try hitting it directly I get a forbidden error. But 8444 maps to nothing, which leads me to believe this is not doing anything.